lib/grpnames.gd

```diff
@@ -16,6 +16,26 @@
 
 #############################################################################
 ##
+#O  IsTrivialNormalIntersection( <G>, <U>, <V> ) . . . . . . . generic method
+##
+##  <ManSection>
+##  <Oper Name="IsTrivialNormalIntersection" Arg="G, U, V"/>
+##
+##  <Description>
+##    For normal subgroups U and V of G, IsTrivialNormalIntersection returns
+##    true if U and V intersect trivially, and false otherwise.
+##    If U and V are not normal subgroups of G, then it may return true
+##    even if they have a nontrivial intersection.
+##  </Description>
+##  </ManSection>
+##
+DeclareOperation( "IsTrivialNormalIntersection",[IsGroup, IsGroup, IsGroup]);
```

Comment:
The formatting here is off, lots of spaces are missing. E.g. this one should be
```
DeclareOperation( "IsTrivialNormalIntersection", [ IsGroup, IsGroup, IsGroup ] )
```
in order to fit in with the rest of the GAP library code.